store/search.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"cloud.google.com/go/bigquery"
+
+	pb "github.com/datacommonsorg/mixer/proto"
+
+	"google.golang.org/api/iterator"
+)
+
+func (s *store) Search(
+	ctx context.Context, in *pb.SearchRequest, out *pb.SearchResponse) error {
+	result := map[string]*pb.SearchResultSection{}
+	tokens := strings.Split(strings.ToLower(in.GetQuery()), " ")
+	qStr := fmt.Sprintf(
+		"SELECT id, type, extended_name FROM `%s`.Instance WHERE type != \"CensusTract\"", s.bqDb)
```

Comment:
PowerPlant and PowerPlantUnit are pretty spammy too.

https://browser.datacommons.org/search?query=Mountain+View